smoketest/graphql2alertlogs_test.go

```diff
@@ -0,0 +1,119 @@
+package smoketest
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/target/goalert/smoketest/harness"
+	"testing"
+)
+
+// TestGraphQL2AlertLogs tests that logs are properly generated for an alert.
+func TestGraphQL2AlertLogs(t *testing.T) {
```

Comment:
Let's drop the 2 from the name and file, it's only temporary while we transition to the new API.